### DISCUSSION TITLE
How can I install Wordpress on Vercel

### DISCUSSION THREAD

**Discussion by helloparent** [vercel/community#4608] [Help]
### Question

Hi,

We are running our website https://helloparent.com on Vercel. We want to integrate Wordpress on Vercel so that we can have our blogs running on  helloparent.com/blogs using wordpress CMS. pls suggest how can it be done?

**Comment by amyegan** [comment]
Hey there! We have a guide to help you use headless WordPress with Next.js and Vercel: https://vercel.com/guides/wordpress-with-vercel

If your site is built with a different framework, you can use a similar headless WordPress strategy. Many helpful folks have published their own guides for consuming WordPress data in other frameworks.

**Comment by yunuserturk** [comment]
There is a step-by-step guide to make it WP static and deploy it to Vercel: https://blog.yunuserturk.com/how-to-host-wordpress-for-free-on-vercel-a-step-by-step-guide/

**Reply by esseifforte** [reply]
HI! thanks for the tutorial!
I followed your guide step by step. when I go to see the site online it gives me a blank page and downloads a file. are there any other steps?

**Reply by yunuserturk** [reply]
Hey, I replied to your question in the main article.
